### PR TITLE
[bep-1279]Fixed keyword arguments for ruby update to 3.1.4 in application-form app

### PR DESCRIPTION
**Pull request by serjpetrenko**
Fixed keyword arguments for ruby update to 3.2.2 in application-form app

https://sohohousedev.atlassian.net/browse/BEP-1279

### CODE REVIEW

**Review by lorint — APPROVED** [review]
This seems to have effectively fixed thoughtbot/paperclip#2673!

**Review by v-lunyov-ep — APPROVED** [review]
is it worth adding some changelog entry in `NEWS`?

**Comment by serjpetrenko** [comment]
> is it worth adding some changelog entry in `NEWS`?

I think `NEWS` was created more for the original gem, as it's forked repository I'm not sure that it's needed here.

File: lib/paperclip/schema.rb [hunk 1]
```diff
@@ -51,7 +51,7 @@ def attachment(*attachment_names)
         attachment_names.each do |attachment_name|
           COLUMNS.each_pair do |column_name, column_type|
             column_options = options.merge(options[column_name.to_sym] || {})
-            column("#{attachment_name}_#{column_name}", column_type, column_options)
+            column("#{attachment_name}_#{column_name}", column_type, **column_options)
```
Comment (ramantsekhanau) [inline lib/paperclip/schema.rb:54]: Did it break our CI or was it just failing on bundling?
Just curious how you find that has to be changed.

**Review by ramantsekhanau — APPROVED** [review]

**Comment by ramantsekhanau** [comment]
Don't forget to push tag `6.1.4` after merge 🙏

File: lib/paperclip/schema.rb (hunk 1 above)
Comment (serjpetrenko) [reply lib/paperclip/schema.rb:54]: It was failed during the `rake db:migrate` on CI, so I found a failed migration with attachement type field and understood that it comes from `paperclip`.

**Pull request merged**